recipes/Kili_fine_tuning_tutorial.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<p>you can also retrieve the assets and labels directly using kili.assets or kili.labels, so that you skip the unzip and json loading/parsing step</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>True. I'll make this change in version 2.0 of the NB</p>